diff --git a/index.js b/index.js
index d7b31ce..ba68c99 100644
--- a/index.js
+++ b/index.js
@@ -1,13 +1,11 @@
 var async = require('async');
 
-// var AWSConfig = {
-//     accessKeyId: '',
-//     secretAccessKey: '',
-//     sessionToken: '',
-//     region: 'us-east-1'
-// };
-
-var AWSConfig = require(__dirname + '/../../cloudsploit-secure/scan-test-credentials.json');
+var AWSConfig = {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+    sessionToken: process.env.AWS_SESSION_TOKEN,
+    region: process.env.AWS_DEFAULT_REGION
+};
 
 var plugins = [
     'iam/rootAccountSecurity.js',
@@ -27,7 +25,7 @@ var plugins = [
     'rds/databaseSecurity.js'
 ];
 
-console.log('CATEGORY\t\tPLUGIN\t\t\t\tTEST\t\t\t\tRESOURCE\t\t\tREGION\t\tSTATUS\tMESSAGE');
+console.log('"CATEGORY","PLUGIN","TEST","RESOURCE","REGION","STATUS","MESSAGE"');
 
 async.eachSeries(plugins, function(pluginPath, callback){
     var plugin = require(__dirname + '/plugins/' + pluginPath);
@@ -46,7 +44,7 @@ async.eachSeries(plugins, function(pluginPath, callback){
                 } else {
                     statusWord = 'UNKNOWN';
                 }
-                console.log(result.category + '\t\t' + result.title + '\t' + result.tests[i].title + '\t' + (result.tests[i].results[j].resource || 'N/A') + '\t' + (result.tests[i].results[j].region || 'Global') + '\t\t' + statusWord + '\t' + result.tests[i].results[j].message);
+                console.log('"' + result.category + '","' + result.title + '","' + result.tests[i].title + '","' + (result.tests[i].results[j].resource || 'N/A') + '","' + (result.tests[i].results[j].region || 'Global') + '","' + statusWord + '","' + result.tests[i].results[j].message + '"');
             }
         }
         callback(err);
